UI/profile.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>Ride Detail</title>
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+</head>
+
+<body>
+    <div class="site-header">
+        <div class="container">
+            <div class="site-header-wrapper">
+                <h2 class="site-title">
+                    Ride My Way
+                </h2>
+                <div class="user-profile-dropdown">
+                    <button class="dropbtn">Micah Oriaso
+                    </button>
+                    <div class="dropdown-content">
+                        <a href="ride-offers.html">Find a ride</a>
+                        <a href="ride-requests.html">Requests</a>
+                        <a href="log-in.html">Log out</a>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="site-body">
+        <div class="main-header">
+            <div class="container">
+                <div class="main-header-wrapper">
+                    <h3>Cynthia West</h3>
+                </div>
+            </div>
+        </div>
+        <div class="container">
+            <div class="form-header">
+                <a href="#" class="primary button pull-right">Update</a>
+            </div>
+            <div class="segment fluid">
+                <div class="grid">
+                    <div class="row">
+                        <div class="eight wide column">
+                            <div class="row"><h3>Profile details</h3></div>
+                            <form id="signup" action="#" method="post">
+                                <div class="form-items">
+                                    <div class="row">
+                                        <div class="eight wide column">
+                                            <div class="form-field">
+                                                <label class="field-label" for="first_name">First Name</label>
+                                                <input type="text" value="Micah" name="first_name" readonly>
+                                            </div>
+                                        </div>
+                                        <div class="eight wide column">
+                                            <div class="form-field">
+                                                <label class="field-label" for="last_name">Last name</label>
```

Comment:
Change to Last Name so that it matches First Name

Reply:
Good catch. It's fixed

![fireshot capture 009 - ride detail - file____home_oriaso_desktop_python_ride-my-way_ui_profile html](https://user-images.githubusercontent.com/20840601/41399487-2afd27fc-6fc3-11e8-9a84-a64e52e455ec.png)

Reply:
I will be attending the session. Looking forward to meeting you there.